pygsti/tools/matrixtools.py

```diff
@@ -1372,6 +1424,9 @@ def _findx(a, inds, always_copy=False):
         return a_inds
 
 
+# TODO: reevaluate the need for this function. It seems like we could just in-line @
```

Comment:
Generally in favor of this. Not a requirement for approval for this PR though, can punt to future.

Reply:
Cool. I'll try to make this change then.